packages/components/src/components/Heading/Heading.module.scss

```diff
@@ -69,6 +69,16 @@
   @include size("l");
   @include size("xl");
   @include size("xxl");
+
+  &:where(.wrap-wrap) {
+    text-wrap: wrap;
+  }
+  &:where(.wrap-nowrap) {
+    text-wrap: nowrap;
+  }
```

Comment:
Should `nowrap` be disallowed? It might break the layout.

> Text does not wrap across lines. It will overflow its containing element rather than breaking onto a new line.

Reply:
Should be removed in favour of a `Elipsis` component.